docker/vttestserver/README.md

```diff
@@ -0,0 +1,16 @@
+## vttestserver docker image
+
+### How to build manually during a release
+
+If for whatever reason the automatic build did not happen for `vttestserver` during a release, or if it has failed,
+there is a way of building it by hand. Here is how it goes:
+
+```bash
+docker login
+
+# we assume in this example that we are releasing v21.0.0-rc1, but replace this by any other tag
+git checkout v21.0.0-rc1
```

Comment:
We should remove the `-rc1` right, since we no longer use that for the branch names? Or are we using tag names?

Reply:
We want to checkout the git tag directly, not the release branch. I updated the comment to be more understandable in 1e950a0.